### PR TITLE
(MODULES-3011) Acceptance Test for Single Session

### PR DESCRIPTION
**Pull request by cowofevil**
Add acceptance tests for PowerShell single session support. Also, I found
some small typos that I fixed as well.
[skip-ci]

### CODE REVIEW

**Comment by cowofevil** [comment]
@Iristyle @jpogran Can you take a look to make sure I covered the scenarios we were interested in for single session?

File: spec/acceptance/exec_powershell_spec.rb [hunk 1]
```diff
@@ -18,17 +18,131 @@
...
+      # Setup the variable
+      apply_manifest(var_leak_setup_pp, :expect_changes => true, :future_parser => FUTURE_PARSER)
+
+      # Test to see if subsequent call sees the variable
+      apply_manifest(var_leak_test_pp, :expect_changes => true, :future_parser => FUTURE_PARSER)
+    end
+
+  end
+
+  describe 'should allow exit from unless' do
```
Comment (Iristyle) [inline spec/acceptance/exec_powershell_spec.rb]: Could we also test the combos for `onlyif`?

Comment (cowofevil) [inline spec/acceptance/exec_powershell_spec.rb]: @Iristyle @glennsarti Added "onlify" tests.

**Comment by cowofevil** [comment]
@glennsarti Updated.

**Pull request merged**